### ISSUE TITLE
Alpha 16?

### ISSUE THREAD

**Issue by unmog** [Karlovsky120/7DaysProfileEditor#10]
First of all I'll just come out and say it. I've know idea how to use Github, it seems to be a confusing mess :p 

Uh I managed to download it before but I wanted to try the new alpha 16 experimental before it became official. So, will the profile editor still work? I dont think so as when I click the exe file it fails to work. I figured Id have to come here to get an update, but I dont know how to Github... >.>; So some quick questions...

1 Is it compatible with the alpha 16 new stuff they added? If not, will it be and when?

2 How do I use this site? No, seriously. It doesnt make sense to me, though it probably makes sense to everyone else. For instance, I see this when I click the main page. http://puu.sh/wjnrM/a4f0198973.png I click the green download or clone button and get a zip file, I look into it and cant even find an exe file to open. I think I remember coming here before and there was some instructions stating to download a config file along with the exe file, but I dont see it anymore... if this is even the same thing I downloaded before. Honestly it all looks the same to me, but Id appreciate some information or a link on a guide on how to use the site. I tried googling it but all the guides seem to be from the perspective of the developer, not just how to find and download/use the thing.

3 While I sincerely wish to learn how to "Github" properly since more and more people are apparently using it and I keep running into this problem of "ease of use" from a consumer standpoint, could you add an easy to find "click here to download the files you need"? I still want to learn how to do it myself, but it would help make it easier for people to come and just get what they need. Some instructions or something would be great.

Otherwise, thanks for making this. Im interested in being able to find and edit my character profile... the new 7d2d system of skills and stuff is very different and I accidentally wasted points. I wish there was just a console command option I could easily use to add the misplaced skill points and fix the mistake but it's being a pain in the butt. However I see the benefit that the editor provides and would like to stay up to date so I can better experiment with the new alpha 16 items and systems added.

**Comment by Karlovsky120** [comment]
Hi!
Thanks for stopping by!

The executables are in the bin folder. Go for the release version and then pick Win32 if unsure which system you have. If it doesn't work, you can try the other versions.

1. The program will support A16, but only after the official release. I'll probably update it within a few days of the release, if I'll have the time.
2. and 3. Well, generally, when talking about programs, the executable is usually within the bin folder. You should have four in the zip you downloaded. I'll add some instructions to the readme when I get the chance.

**Comment by MarvinFS** [comment]
Hi, thank you for your efforts first of all, second of all, a16 compatibility is like possible working or not working at all, as im unable to run it now, may be things changed in exe file so tool can't parse it properly - i get failed to load asset files unable to read beyond the end of the stream error. tried all 4 bin files (debug, stable, 32bit 64 bit)

**Comment by Karlovsky120** [comment]
That is probably because A16 support hasn't been implemented yet. I'll do it once A16 is official, at the same time dropping support for A15.

**Comment by CynicalVirtuoso** [comment]
I could do a new branch and pull request for once changes I have done for A16 if Karlovsky wants. I just went over the new save file to see what had changed to break it and the only thing I noticed is a new byte has been inserted in between "finishTime" and "objectiveCount" so simply grabbing that random byte out of the reader should be sufficient enough to allow it to work again. I will read through the code later to see if it is even meaningful at all. 

File: Quest.cs
`
finishTime = new Value<ulong>(reader.ReadUInt64());

int randomByte = reader.ReadByte();

//num
int objectiveCount = reader.ReadByte();
`
